Experimental/DistributedAuthoritySample/Assets/Scripts/Input/AvatarInputs.cs

```diff
@@ -0,0 +1,76 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace com.unity.multiplayer.samples.distributed_authority.input
+{
+    public class AvatarInputs : MonoBehaviour
```

Comment:
1. Does this (and its method) need to be public, or would it be good if they were  internal?
2. if they must be public, they need to be documented (///)

Reply:
Same as above!